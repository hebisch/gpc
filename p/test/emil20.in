first line
second line, with no terminator