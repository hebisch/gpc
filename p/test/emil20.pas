program BufferedInput (Output, F);

var
  F: bindable Text;
  Buffer: String (100);

begin
  Reset (F, '');
{$extended-pascal}
  while not EOF (F) do begin
    while not EOLn (F) do begin
      Read (F, Buffer);
      WriteLn ('read: `', Buffer, '''')
    end;
    WriteLn ('(end of line)');
    ReadLn (F)
  end;
  WriteLn ('(end of file)')
end.
