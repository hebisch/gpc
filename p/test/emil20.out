read: `first line'
(end of line)
read: `second line, with no terminator'
(end of line)
(end of file)
